python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -760,13 +768,7 @@ def job_finalisation(job, updated_dirac_status):
                     job.updateStatus('running')
                 if job.status in ['completed', 'killed', 'removed']:
                     break
-                # make sure proxy is valid
-                if DiracBase.checkDiracProxy():
-                    # perform finalisation
-                    DiracBase._internal_job_finalisation(job, updated_dirac_status)
-                else:
-                    # exit gracefully
-                    logger.warning("Cannot process job: %s. DIRAC monitoring has been disabled. To activate your grid proxy type: \'gridProxy.renew()\'" % job.fqid)
+                DiracBase._internal_job_finalisation(job, updated_dirac_status)
```

Comment:
Does this still exit gracefully or does it demand a user give proxy details because the monitoring system has found a grid job?
This can be annoying when wanting to fire up ganga for a non-grid related task when the repo may have large numbers of jobs known to the user not to need updating.